index.js

```diff
@@ -630,10 +630,16 @@ class Readable extends Stream {
   resume () {
     this._duplexState |= READ_RESUMED
     this._readableState.updateNextTick()
+    return this
+  }
+
+  isPaused () {
+    return (this._duplexState & READ_PAUSED) !== 0 && (this._duplexState & READ_RESUMED) === 0
   }
```

Comment:
i think we should make this a static method like the isBackpressured ones, to avoid adding more APIs people might overwrite by accident, when extending

Reply:
You only need to check if the resumed bit is 0 here also. (duplexState & READ_RESUMED) === 0

Reply:
I changed it to a static method but I am not sure what you mean with the `READ_RESUMED` part. It looks like it already does what you mean?